crates/cheatcodes/Cargo.toml

```diff
@@ -40,9 +40,12 @@ alloy-signer-local = { workspace = true, features = [
     "keystore",
 ] }
 parking_lot.workspace = true
-alloy-consensus = { workspace = true, features = ["k256"] }
 alloy-network.workspace = true
-alloy-rlp.workspace = true
+# Note(zk): This is now commented out because we are using the alloy-concencus crate within the strategy
```

Comment:
No need to be overly specific, just something like:
`NOTE(zk): dependency moved to strategy crate` would be sufficient I think